### DISCUSSION TITLE
@mentions in microblog posts

### DISCUSSION THREAD

**Discussion by scottsweb** [beakerbrowser/beaker#1758] [General] [answered]
Just getting acquainted with the new Beaker - lots of great improvements, thanks for all your hard work!

I noticed that some microblog posts have @mentions to other posts - are these added by hand as custom markdown links at the moment? and does @mentioning someone notify the recipient in some way?

**Comment by pfrazee** [comment, marked as answer]
They're just markdown links people are adding manually. If somebody were interested in making an app with notifications, they could probably do it by scanning posts for those links!

**Reply by scottsweb** [reply]
Thanks for the speedy reply, just wanted to check I wasn't missing something. Will tinker some more!
